http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/HttpVersionClientResource.java

```diff
@@ -0,0 +1,149 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.time.Duration;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.resteasy.reactive.client.impl.ClientResponseImpl;
+
+import io.quarkus.ts.http.restclient.reactive.HttpVersionClient;
+import io.quarkus.ts.http.restclient.reactive.HttpVersionClientWithConfigKey;
+import io.quarkus.ts.http.restclient.reactive.HttpsVersionClientWithConfigKey;
+import io.vertx.core.http.HttpVersion;
+
+@Path("/http2")
+public class HttpVersionClientResource {
+
+    public static final String WRONG_HTTP_VERSION = "The HTTP version should be HTTP_2 but is ";
```

Comment:
```suggestion
    public static final String WRONG_HTTP_VERSION = "The HTTP version should be HTTP/2 but is ";
```

Reply:
Now I'm thinking of it, you probably used actual response type value as they use underscore, fine then.